tests/api_tests/nodes/test_views.py

```diff
@@ -534,6 +534,7 @@ def test_create_project_invalid_title(self):
         res = self.app.post_json_api(self.url, project, auth=self.user_one.auth, expect_errors=True)
         assert_equal(res.status_code, 400)
         assert_equal(res.json['errors'][0]['detail'], 'Title cannot exceed 200 characters.')
+        assert_equal(res.json['errors'][0]['source'], {}, 'Source cannot be none per JSON API Spec')
```

Comment:
This provides three arguments to `assert_equal`. 

If the third argument is indeed a custom error message to make the test output more readable, please make sure to add the `msg=` to the third argument. 
https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertEqual

(Treating kwargs as positional brings only pain. If the call signature is specific, you should be too)